ipa-core/src/bin/report_collector.rs

```diff
@@ -329,6 +358,94 @@ fn write_ipa_output_file(
     Ok(())
 }
 
+fn write_hybrid_output_file(
```

Comment:
this looks oddly familiar, is it possible to re-use the same code and avoid copy paste here?

Reply:
same old problem with Oprf and Hybrid. we can either spend time supporting both or just delete the other when we remove Oprf.